src/pages/components/dropdown/usage.mdx

```diff
@@ -25,18 +31,27 @@ import items from "../../../data/components/dropdown.js";
 
 ## Overview
 
-Dropdowns present a list of options that can be used to filter or sort existing content.
+There are three different types of dropdowns that support various kinds of functionality—dropdown, multiselect dropdown, and combobox.
+
+### Types
+
+| Type                        | Purpose                                                                                                                                            |
+| --------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Dropdown](#dropdown)       | Allows the user to select one option from a list.                                                                                                  |
+| [Multiselect](#multiselect) | Allows the user to select multiple options from a list and filter.                                                                                 |
+| [Combobox](#combobox)       | Allows the the user to make a selection from a predefined list of options either by typing a value directly or by selecting a value from the list. |
+
+#### When to use
 
-They can also be used as menus. Their use as menus can be seen in [tabs](/components/tabs/usage). At a smaller screen size, the tabs collapse into a dropdown.
+Dropdowns can be used in forms on full pages, in modals, or on side panels. The dropdown component is used to filter or sort contents on a page. It is a stylized version of the select component, and can be styled as needed.
 
-### Dropdown versus Select
+#### When not to use
 
-While the dropdown and select components look similar, they have different functions.
+- It is best practice not to use a dropdown if there are two or less options to choose from. In this case, use a [radio button](https://www.carbondesignsystem.com/components/radio-button/code) group instead.
 
-- Use the dropdown component to filter or sort content on a page.
-- Use the select component inside a form where users are selecting from a list of options and submitting data.
+- Do not nest dropdowns or use them to display overly complex information. Keep option selections as straight forward as possible.
 
-Another important difference between the two components is the underlying code. The dropdown component is styled to match the design system, while the select component's appearance will be determined by the browser being used.
+- If your user is selecting from a list of options and submitting data inside a form, use the [select component](https://www.carbondesignsystem.com/components/select/usage) instead.
```

Comment:
This may be a misunderstanding, but this seems to conflict with a section later on under Related:

> Use a dropdown component in forms, to select multiple options at a time and to filter or sort content on a page.

Reply:
I know that select dropdowns are typically used when submitting data in a form, but you could also use the stylized dropdown if you wanted to right? Is there any benefit from using select over dropdown in a form? I think I just need to make a clearer argument on the uses.